### DISCUSSION TITLE
Q&A탭 API명세서

### DISCUSSION THREAD

**Discussion by sokft** [ttakkeun/Front_iOS#45] [Q&A] [answered]
Q&A탭 관련 기능명세서도 없고, API 명세서도 없는데 질문과 답변은 그냥 정해진 질문과 답 나오도록 하면 되는 건가요?

**Comment by JEONG-J** [comment, marked as answer]
기능 명세서랑 API 명세서 없이도 만들 수 있지!
1) FAQ는 서버가 전혀 필요 없는 뷰라 온전히 프론트로만 구현하는 거
2) 프론트에서 어떻게 구현하는가? -> 데이터 모델 만들고 질문에 해당하는 JSON 넣어서 파싱하도록 한다
3) 질문이 FAQ 질문과 내용이 아직 미완성이긴 하지만 이거는 추후 완성되면 JSON의 값만 바꾸면 되는 문제
4) 본인이 뷰를 만들면서 JSON에 그럴싸한 질문과 답변 아무거나 넣어서 디자인된거 처럼 나오도록 하고 애니메이션 넣기만하면 나중에 JSON만 바꾸더라도 뷰는 유지되기 때문에 아무 문제 없음
5) 기능 명세서 필요 없는 이유는 뷰가 아주 단순하고 API도 필요 없기 떄문이다.
6) TIPS의 경우 API가 필요 하겠지만 이건 FAQ만드는 동안 만들어지기 때문에 FAQ부터 끝낼것 대신 TIPS도 만들어야 하기 때문에 세그먼트 버튼 생각하고 배치시킬 것
